src/challengepaths/2015/CommunityService.ts

```diff
@@ -0,0 +1,352 @@
+  new Requirement(["Mysticality"], {})
+);
+
+export const Moxie = new Test(
+  4,
+  "Feed Conspirators",
+  statTestPredictor($stat`moxie`),
+  new Requirement(["Moxie"], {})
+);
+
+export const FamiliarWeight = new Test(
+  5,
+  "Breed More Collies",
+  () =>
+    60 - Math.floor((familiarWeight(myFamiliar()) + weightAdjustment()) / 5),
+  new Requirement(["Familiar Weight"], {})
+);
+
+export const WeaponDamage = new Test(
+  6,
+  "Reduce Gazelle Population",
+  () => {
+    const weaponPower = getPower(equippedItem($slot`weapon`));
+    const offhandPower =
+      toSlot(equippedItem($slot`off-hand`)) === $slot`weapon`
+        ? getPower(equippedItem($slot`off-hand`))
+        : 0;
+    const familiarPower =
+      toSlot(equippedItem($slot`familiar`)) === $slot`weapon`
+        ? getPower(equippedItem($slot`familiar`))
+        : 0;
+    const songDamage =
+      SongBoom.song() === "These Fists Were Made for Punchin'" ? myLevel() : 0;
+
+    // mafia does not currently count swagger
+    const denominator = have($effect`Bow-Legged Swagger`) ? 25 : 50;
+    return (
+      60 -
+      Math.floor(
+        (getModifier("Weapon Damage") -
+          0.15 * (weaponPower + offhandPower + familiarPower) -
+          songDamage) /
+          denominator +
+          0.001
+      ) -
+      Math.floor(getModifier("Weapon Damage Percent") / denominator + 0.001)
+    );
+  },
+  new Requirement(["Weapon Damage", "Weapon Damage Percent"], {})
+);
+
+export const SpellDamage = new Test(
+  7,
+  "Make Sausage",
+  () => {
+    const dragonfishDamage =
+      myFamiliar() === $familiar`Magic Dragonfish`
+        ? numericModifier(
+            $familiar`Magic Dragonfish`,
+            "Spell Damage Percent",
+            familiarWeight($familiar`Magic Dragonfish`) + weightAdjustment(),
+            $item`none`
+          )
+        : 0;
+    return (
+      60 -
+      Math.floor(getModifier("Spell Damage") / 50 + 0.001) -
+      Math.floor(
+        (getModifier("Spell Damage Percent") - dragonfishDamage) / 50 + 0.001
+      )
+    );
+  },
+  new Requirement(["Spell Damage", "Spell Damage Percent"], {})
+);
+
+export const Noncombat = new Test(
+  8,
+  "Be a Living Statue",
+  () => {
+    const noncombatRate = -1 * getModifier("Combat Rate");
+    const unsoftcappedRate =
+      noncombatRate > 25 ? 25 + (noncombatRate - 25) * 5 : noncombatRate;
+    return 60 - 3 * Math.floor(unsoftcappedRate / 5);
+  },
+  new Requirement(["-combat"], {})
+);
+
+export const BoozeDrop = new Test(
+  9,
+  "Make Margaritas",
+  () => {
+    const familiarItemDrop = numericModifier(
+      myFamiliar(),
+      "Item Drop",
+      familiarWeight(myFamiliar()) + weightAdjustment(),
+      equippedItem($slot`familiar`)
+    );
+
+    //Champagne doubling does NOT count for CS, so we undouble
+    const denominator =
+      haveEquipped($item`broken champagne bottle`) &&
+      get("garbageChampagneCharge") > 0
+        ? 60
+        : 30;
```

Comment:
1. It doesn't!
2. This actually _divides_ by 2, because mafia counts champagne and kol doesn't.

Would you rather I multiply by `stuff ? 0.5 : 1`, or divide by `stuff ? 2 : 1`?